source/mainnet/net/nodes/node-runner-service-configuration.rst

```diff
@@ -5,7 +5,9 @@
 Concordium Windows node runner service configuration
 ====================================================
 
-The Concordium Node Runner Service ("service" for short) is configured using a TOML file. This file is typically located at ``C:\ProgramData\Concordium\Node Runner\nodes.toml.`` (The path on your system is determined by the ``Config`` value in the registry key ``HKEY_LOCAL_MACHINE\SOFTWARE\Concordium\Node Runner``.)
+You can change the mainnet or testnet node configuration with the Configure Concordium Node Service. The Concordium Node Runner Service ("service" for short) is configured using a TOML file. This file is typically located at ``C:\ProgramData\Concordium\Node Runner\nodes.toml.`` (The path on your system is determined by the ``Config`` value in the registry key ``HKEY_LOCAL_MACHINE\SOFTWARE\Concordium\Node Runner``.)
+
+To run the app, search for *configure concordium node* in the **Search** bar, and then select **Configure Concordium Node Service**. If you see a message saying *Do you want to allow this app to make changes to your device?*, select **Yes**. The configuration file opens in Notepad.
```

Comment:
I think this last part "opens in Notepad" is technically incorrect. If they have set some other editor to open `toml` files then it'll be that app that opens.

But it's probably better to leave it like this. If people have editors to edit TOML files then they'll know what's going on I imagine.